src/FromFile.jl

```diff
@@ -1,8 +1,9 @@
 module FromFile
 
-export @from
+export @from, @from_url
 
 using Requires
+using Downloads
```

Comment:
Can we switch to `import`? This kind of explict-ness if part of what FromFile.jl is about, after all 😄

Reply:
I changed to `using Downloads: download`, I think good practice is to avoid `import` as much as possible in Julia, as it will cause implicit overloading of generic functions